lib/usd/ui/USDImportDialog.cpp

```diff
@@ -72,15 +79,20 @@ USDImportDialog::USDImportDialog(const std::string& filename, const ImportData*
 	fUI->treeView->setSelectionMode(QAbstractItemView::SingleSelection);
 	QObject::connect(fUI->treeView, SIGNAL(clicked(const QModelIndex&)), this, SLOT(onItemClicked(const QModelIndex&)));
 	QObject::connect(fUI->actionReset_File, SIGNAL(triggered(bool)), this, SLOT(onResetFileTriggered()));
+	QObject::connect(fUI->actionHelp_on_Hierarchy_View, SIGNAL(triggered(bool)), this, SLOT(onHierarchyViewHelpTriggered()));
 
 	QHeaderView* header = fUI->treeView->header();
 
 	header->setStretchLastSection(true);
 	header->setDefaultAlignment(Qt::AlignLeft | Qt::AlignVCenter);
 
+	fItemDelegate = std::unique_ptr<ItemDelegate>(new ItemDelegate(fUI->treeView));
+	QObject::connect(fItemDelegate.get(), SIGNAL(variantModified()), 
+		fTreeModel.get(), SLOT(updateModifiedVariantCount()));
+
 	// Set our item delegate on the treeview so we can customize the drawing of
 	// the variant sets.
-	fUI->treeView->setItemDelegate(new ItemDelegate(fUI->treeView));
```

Comment:
I think this was a memory leak.

Reply:
Yeah I think you are right. In the docs it says that "QAbstractView does not take ownership of delegate". Thanks for catching this.